src/c++/perf_analyzer/genai-perf/genai_perf/profile_data_parser/llm_profile_data_parser.py

```diff
@@ -218,6 +218,8 @@ def _get_openai_input_text(self, req_inputs: dict) -> str:
             return payload["messages"][0]["content"]
         elif self._response_format == ResponseFormat.OPENAI_COMPLETIONS:
             return payload["prompt"]
+        elif self._response_format == ResponseFormat.HUGGINGFACE_RANKINGS:
```

Comment:
I thought the rankings support was only for request level metrics?
Do we ever hit this if case?

Reply:
Good catch! I did not know that. Removed and everything still runs successfully:
![image](https://github.com/triton-inference-server/client/assets/58150256/3c6e504b-33ca-4e31-9387-297a60b230e2)